backend/src/dao/ShoutoutsDao.ts

```diff
@@ -1,20 +1,21 @@
+import { v4 as uuidv4 } from 'uuid';
 import { memberCollection, shoutoutCollection } from '../firebase';
 import { Shoutout, DBShoutout } from '../types/DataTypes';
+import { NotFoundError } from '../utils/errors';
 import { getMemberFromDocumentReference } from '../utils/memberUtil';
 
 export default class ShoutoutsDao {
   static async getAllShoutouts(): Promise<Shoutout[]> {
+    const shoutoutRefs = await shoutoutCollection.get();
     return Promise.all(
-      await shoutoutCollection.get().then((shoutoutRefs) =>
-        shoutoutRefs.docs.map(async (doc) => {
-          const dbShoutout = doc.data() as DBShoutout;
-          const giver = await getMemberFromDocumentReference(dbShoutout.giver);
-          return {
-            ...dbShoutout,
-            giver
-          };
-        })
-      )
+      shoutoutRefs.docs.map(async (doc) => {
+        const dbShoutout = doc.data() as DBShoutout;
+        const giver = await getMemberFromDocumentReference(dbShoutout.giver);
+        return {
+          ...dbShoutout,
+          giver
+        };
+      })
```

Comment:
What's the purpose of this change?

Reply:
Makes the function more consistent with how other `getAll()` functions are structured. For example in `CandidateDeciderDao.ts `